backend/src/test/java/heartbeat/service/jira/JiraServiceTest.java

```diff
@@ -713,7 +714,8 @@ void shouldGetCardsWhenCallGetStoryPointsAndCycleTimeGivenStoryPointKeyFromEnvir
 		StoryPointsAndCycleTimeRequest storyPointsAndCycleTimeRequest = STORY_POINTS_FORM_ALL_DONE_CARD().build();
 		storyPointsAndCycleTimeRequest.setOverrideFields(jiraBoardSetting.getOverrideFields());
 		CardCollection cardCollection = jiraService.getStoryPointsAndCycleTimeAndReworkInfoForDoneCards(
-				storyPointsAndCycleTimeRequest, jiraBoardSetting.getBoardColumns(), List.of("Zhang San"), "");
+				storyPointsAndCycleTimeRequest, jiraBoardSetting.getBoardColumns(), List.of("Zhang San"), "",
+				ZoneId.of("Asia/Shanghai"));
```

Comment:
Duplicated code

Reply:
I don't understand, I don't find any duplicated code. if you say the ```jiraService.getStoryPointsAndCycleTimeAndReworkInfoForDoneCards(
				storyPointsAndCycleTimeRequest, jiraBoardSetting.getBoardColumns(), List.of("Zhang San"), "",
				ZoneId.of("Asia/Shanghai"))``` is duplicated, the params is different.